testdata/p4_16_samples_outputs/pragma-string.p4-stderr

```diff
@@ -1 +1,17 @@
+pragma-string.p4(20): [--Wwarn=unused] warning: 'original' is unused
+const bit b = 1;
+          ^
+pragma-string.p4(23): [--Wwarn=unused] warning: 'string \" with \" quotes' is unused
+const bit c = 1;
+          ^
+pragma-string.p4(27): [--Wwarn=unused] warning: 'string with
+newline' is unused
+const bit d = 1;
```

Comment:
These warnings seem wrong (or at least in the wrong place) -- it is apparently complaining about the strings in the `@pragma` on the preceding line being unused?

Reply:
This doesn't seem to be related to this PR. These files seem out of sync as they were added when I ran `P4TEST_REPLACE=1 ninja check-*`

Reply:
I can move these changes to a separate PR if you prefer they be updated separately